articles/141_static_remapping.md

```diff
@@ -0,0 +1,435 @@
+---
+layout: default
+title: Remapping Names
+permalink: articles/static_remapping.html
+abstract:
+  Topics, parameters, and services are identified by [Names](http://wiki.ros.org/Names). Names are hard coded in ROS nodes, but they can be changed at runtime through remapping. Without remapping every instance of a node would require changes in code. This article describes the requirements, rationale, and mechanisms for remapping names in ROS 2.
+author: '[Shane Loretz](https://github.com/sloretz)'
+published: true
+---
+
+- This will become a table of contents (this text will be scraped).
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+## Why remap names?
+Remapping names allows reusing the same node executable in different parts of the system.
+A robot that has multiple sensors of the same type could launch multiple instances of the same node with outputs remapped to different topics.
+
+```
+            +-------------+
+ (Lidar 1)--> node inst 1 +-->/head_scan
+            +-------------+
+
+            +-------------+
+ (Lidar 2)--> node inst 2 +-->/base_scan
+            +-------------+
+```
+
+
+## Structure of a Name
+Names are hierarchical strings delineated by `/`.
+If they begin with `/` they are said to be **global** or **Fully Qualified Names** (FQN).
+Otherwise they are said to be **relative**.
+Relative names are resolved to FQN prior to being used.
+
+**Example names:**
+
+- `/foo`
+- `/foo/bar`
+- `foo/bar/baz`
+- `foo/bar`
+- `bar`
+
+Each part of a name between a slash is a **token**.
```

Comment:
Wording:

> between a slash

Should this be?

> between slashes

(since it can't be "between" a single slash but only between two slashes.)